zerver/tests/test_audit_log.py

```diff
@@ -51,3 +51,15 @@ def test_change_email(self):
         self.assertEqual(RealmAuditLog.objects.filter(event_type='user_email_changed',
                                                       event_time__gte=now).count(), 1)
         self.assertEqual(email, user.email)
+
+class TestChangeAvatarFields(ZulipTestCase):
+    def test_change_avatar_source(self):
+        # type: () -> None
+        realm = get_realm('zulip')
+        now = timezone.now()
+        user = do_create_user('email', 'password', realm, 'full_name', 'short_name')
+        avatar_source = '/img/1.jpg'
```

Comment:
Your bug is because valid values for `avatar_source` are things like `AVATAR_FROM_GRAVATAR = G`.  Read the `zerver/models.py` code for this field and you'll understand.